server/src/main/java/org/apache/iotdb/db/engine/TsFileMetricManager.java

```diff
@@ -32,6 +32,10 @@ public class TsFileMetricManager {
   private final AtomicInteger seqFileNum = new AtomicInteger(0);
   private final AtomicInteger unseqFileNum = new AtomicInteger(0);
 
+  private final AtomicInteger modFileNum = new AtomicInteger(0);
+
+  private final AtomicLong modFileSize = new AtomicLong(0);
```

Comment:
Consider whether the size may too big?

Reply:
It's AtomicLong, the size of mods files can be up to 8388608 TB.